README.md

```diff
@@ -3627,19 +3627,20 @@ resource cleanup when possible.
 <sup>[[link](#percent-s)]</sup>
 
 * <a name="percent-literal-braces"></a>
-  Prefer `()` as delimiters for all `%` literals, except `%r`. Since parentheses
-  often appear inside regular expressions in many scenarios a less common
-  character like `{` might be a better choice for a delimiter, depending on the
-  regexp's content.
+  Prefer `()` as delimiters for all `%` literals, except `%r` and Array 
```

Comment:
Just a tiny remark - should Array be capitalised? I guess it depends on whether we're discussing the class or arrays in general.